src/helpers/notification_helpers.ts

```diff
@@ -1,45 +1,143 @@
+        } = cache_manager.getObjectData("nx-settings");
+        const data = { UserName: monogoto.UserName, Password: monogoto.Password };
+
+        const response = await axios({
             method: "post",
-            maxBodyLength: Infinity,
-            url: "https://api.multisend.co.il/v2/sendsms",
-            data: data,
+            url: `https://console.monogoto.io/Auth`,
+            data,
+        });
+        return response.data;
+    } catch (error) {
+        throw `login_to_monogoto failed: ` + error;
+    }
+};
+
+const send_iccid_sms: SmsFunction = async (number, text) => {
+    const {
+        sms_provider: { monogoto },
+    } = cache_manager.getObjectData("nx-settings");
+    const monogoto_auth = await login_to_monogoto();
+    const data = { Message: text, From: monogoto.from };
+    const response = await axios({
+        method: "post",
+        url: `https://console.monogoto.io/thing/ThingId_ICCID_${number}/sms`,
+        data: data,
+        headers: {
+            Authorization: `Bearer ${monogoto_auth.token}`,
+            apikey: monogoto_auth.CustomerId,
+        },
+    });
+    if (response.status !== 200) {
+        throw `request to monogoto status: ${response.status}`;
+    }
+    await add__new_sms_to_db(number, text, "monogoto", response.data);
+    logger.log("send_iccid_sms. message sent successfully", { number, text, response: response.data });
+};
+
+export const send_sms = async (number: string, text: string, entity_for_audit: string) => {
+    try {
+        let service = "multisend";
+        const send = async () => {
+            if (isIccid(number)) {
+                service = "monogoto";
+                return await send_iccid_sms(number, text);
+            }
+            if (isInternationalPhoneNumber(number)) {
+                service = "twilio";
+                return send_international_sms(number, text);
+            }
+            return await send_local_sms(number, text);
         };
-        const response = await axios(config);
-        if (response.status !== 200) {
-            throw `http request to multisend status ${response.status}`;
-        }
-        if (!response.data.success) {
-            throw `http request to multisend error ${JSON.stringify(response.data.error)}`;
-        }
-        if (!response.data.success) {
-            throw response.data.error;
-        }
-        await add_audit_record("send_sms", entity_for_audit, {
-            destination: phone_number,
+        await send();
+        await add_audit_record("send_sms", entity_for_audit || "general", {
+            destination: number,
             message: text,
+            service,
         });
     } catch (error) {
         logger.error(`${entity_for_audit}, send_sms failed:`, error);
         throw `${entity_for_audit}, send_sms failed: ` + error;
     }
 };
 
+const add__new_sms_to_db = async (recipient: string, content: string, service: string, external_id: string) => {
```

Comment:
a) any special reason to use "__" ?
b) make better name: add__new_sms_to_db -> add_sms_to_db or keep_sms. 
      - No need to use words "new" and "db"
      - need to use "outgoing" - because we also have incoming SMS

Reply:
done